code/game/atoms.dm

```diff
@@ -18,6 +18,8 @@
 	//Value used to increment ex_act() if reactionary_explosions is on
 	var/explosion_block = 0
 
+	//overlays that should remain on top and not normally be removed, like c4.
+	var/list/priority_overlays = list()

```

Comment:
this should be null and your helper procs should initialise the list when it's first grabbed or added to